projects/steering/src/steering_can.c

```diff
@@ -7,16 +8,18 @@
 #include "steering_digital_input.h"
 #include "steering_events.h"
 
+static bool turning = false;
+
 StatusCode steering_can_process_event(Event *e) {
   switch (e->id) {
     case STEERING_INPUT_HORN_EVENT:
       CAN_TRANSMIT_HORN((EEHornState)e->data);
       break;
     case STEERING_DRL_1_EVENT:

```

Comment:
Could you rename `DRL_1` to `DRL_ON` and `DRL_2` to `DRL_OFF` throughout steering+smoke_steering since that's what their purpose is now?

Reply:
Sure